backend/scripts/send-tec-notifications.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-console */
+import admin from 'firebase-admin';
+import sendMail from './utils';
+import { configureAccount } from '../src/utils/firebase-utils';
+
+const serviceAcc = require('../resources/idol-b6c68-firebase-adminsdk-h4e6t-40e4bd5536.json');
+
+admin.initializeApp({
+  credential: admin.credential.cert(configureAccount(serviceAcc, 'prod')),
+  databaseURL: 'https://idol-b6c68.firebaseio.com',
+  storageBucket: 'gs://cornelldti-idol.appspot.com'
+});
+
+const db = admin.firestore();
+
+const EMAIL_SUBJECT = '[ACTION REQUIRED] Pending TEC Requests';
+const EMAIL_BODY =
+  'Hey there from the IDOL team!\n\nThere are DTI members who have submitted TEC requests that are currently pending. Please visit https://idol.cornelldti.org/admin/team-events to review the requests.\n\nThank you!';
+
+const main = async () => {
+  console.log('Reading all team event credit requests...');
+  const numPendingRequests = (
+    await db.collection('team-event-attendance').where('status', '==', 'pending').get()
+  ).docs.length;
```

Comment:
What happened to the count aggregation :( 
https://firebase.google.com/docs/firestore/query-data/aggregation-queries#use_the_count_aggregation

Reply:
Sorry, I added that back. Just needed to fix some type issues, and a few version bumps.